lib/rspec/rails/feature_check.rb

```diff
@@ -1,3 +1,8 @@
+begin
+  require "active_job"
+rescue LoadError # rubocop:disable Lint/HandleExceptions
+end
+
```

Comment:
I feel like if this isn't necessary normally, it shouldn't be here, can it be in the features with the feature check?